lib/routes/dw/utils.ts

```diff
@@ -0,0 +1,168 @@
+                case 'hi':
+                    return 'तस्वीर';
+                case 'id':
+                    return 'Foto';
+                case 'sw':
+                    return 'Picha';
+                case 'mk':
+                    return 'Фотографија';
+                case 'ps':
+                    return 'انځور';
+                case 'fa-ir':
+                    return 'عکس';
+                case 'pl':
+                    return 'Zdjęcie';
+                case 'pt-002':
+                    return 'Foto';
+                case 'pt-br':
+                    return 'Foto';
+                case 'ro':
+                    return 'Imagine';
+                case 'ru':
+                    return 'Фото';
+                case 'sr':
+                    return 'Foto';
+                case 'es':
+                    return 'Imagen';
+                case 'tr':
+                    return 'Fotoğraf';
+                case 'uk':
+                    return 'Фото';
+                case 'ur':
+                    return 'تصویر';
+                default:
+                    return 'Image';
+            }
+        default:
+            return word;
+    }
+};
+
+const processHtml = ($: CheerioAPI, contentLinks) => {
+    $('img').each((_, elem) => {
+        try {
+            const id = $(elem).attr('data-id');
+            const contentLink = contentLinks.find((item) => String(item.targetId) === id);
+            $(elem).attr({
+                title: contentLink?.name,
+                alt: contentLink?.description,
+                src: `https://static.dw.com/image/${id}_${formatId}.jpg`,
+            });
+            $(elem).removeAttr('style');
+        } catch {
+            // no-empty
+        }
+    });
+    $('video').each((_, elem) => {
+        try {
+            $(elem).attr('poster', $(elem).attr('data-posterurl'));
+        } catch {
+            // no-empty
+        }
+    });
+    $('iframe').each((_, elem) => {
+        try {
+            $(elem).attr('src', $(elem).attr('data-src'));
+        } catch {
+            // no-empty
+        }
+    });
+    $('svg').remove(); // svg will screw up in a lot of rss readers
+};
+
+const processContent = (item, content) => {
+    const $text = load(content.text);
+    processHtml($text, content.contentLinks);
+    const liveblog =
+        item.type === 'liveblog' && content.posts
+            ? art(path.join(__dirname, 'templates/liveblog.art'), {
+                  posts: content.posts.map((post) => {
+                      const $post = load(post.text);
+                      processHtml($post, content.contentLinks);
+                      post.text = $post.html();
+                      return post;
+                  }),
+              })
+            : undefined;
+    const video =
+        item.type === 'video' && content.hlsVideoSrc
+            ? art(path.join(__dirname, 'templates/video.art'), {
+                  hlsVideoSrc: content.hlsVideoSrc,
```

Comment:
You can craft the mp4 URL from the HLS URL. For example, you got 
```
https://hlsvod.dw.com/i/dwtv_video/flv/igs/igs20241115_gesamtHD_,AVC_480x270,AVC_512x288,AVC_640x360,AVC_960x540,AVC_1280x720,AVC_1920x1080,.mp4.csmil/master.m3u8
```
in the first item of https://github.com/DIYgod/RSSHub/pull/17575#issuecomment-2478854266 and the corresponding mp4 URL is
```
https://tvdownloaddw-a.akamaihd.net/dwtv_video/flv/igs/igs20241115_gesamtHD_AVC_1920x1080.mp4
```
so user can enjoy the video as HLS aren't really playable without the aid of external JS lib.

Reply:
I added a function to brute force convert the hls url to mp4 url with replace(), not an elegant solution but it works (at least until the url format changes)